packages/template-retail-react-app/app/hooks/use-product-view-modal.test.js

```diff
@@ -92,30 +92,6 @@ describe('useProductViewModal hook', () => {
         })
     })
 
-    test("update product's related url param when the product content is shown", () => {
```

Comment:
This is just a side-effect for how state is manage. Even the modal uses query params

Reply:
> Would it make sense for this test to move to app/components/product-view/index.test.js?

I've deleted that test because I've removed the corresponding code here: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1310#discussion_r1241817609

Reply:
> I still don't fully understand why the query params get updated on the cart page

The query params are needed for the swatch selection to fully work. Without the params, users click on the swatches and nothing would happen visually.

Let's contrast it with the PDP. On the PDP, the query params are necessary for these things:
- swatch selection
- AND also for the bookmark-able url (so you can easily share the page that already has its variants chosen)

However, on the Cart page, the query params are necessary for 1 thing only:
- swatch selection
- (no bookmark-able url for the Cart... that's why the page url would get reset every time you close the modal)